sherpa/models/basic.py

```diff
@@ -1640,8 +1652,10 @@ def set_center(self, xpos, ypos, *args, **kwargs):
     def guess(self, dep, *args, **kwargs):
         xpos, ypos = guess_position(dep, *args)
         ampl = guess_amplitude2d(dep, *args)
```

Comment:
I think we would want this to be used to guess the ampl/norm/whatever parameter.

Reply:
Not sure what you mean here, doesn't the following existing code:
```
        # Apply normalization factor to guessed amplitude
        norm = (numpy.pi / _gfactor) * self.fwhm.val * self.fwhm.val * \
            numpy.sqrt(1.0 - (self.ellip.val * self.ellip.val))
        for key in ampl.keys():
            if ampl[key] is not None:
                ampl[key] *= norm
        param_apply_limits(ampl, self.ampl, **kwargs)
```
attempts at guessing the ```ampl``` parameter?

Reply:
I think I must have just missed that part, as it was separated by a pesky blank line, which threw me off. He guesses wildly.